python/tchannel/thrift/scheme.py

```diff
@@ -28,7 +28,7 @@
 from .. import scheme
 
 
-class ThriftArgScheme(scheme.ArgScheme):
+class ThriftArgScheme(scheme.RawArgScheme):
```

Comment:
I disagree. No need for inheritance here. If you want to define an interface  (`abc` in python), that's a different story.